sdk/packages/react/src/internal/api.ts

```diff
@@ -22,8 +22,9 @@ export async function fetchJSON(
   const json = await res.json()
 
   if (!res.ok) {
-    if (!isJSONError(json.error)) new Error(`${res.status} ${res.statusText}`)
-    throw json.error
+    if (isJSONError(json)) throw json
+    if (isJSONError(json.error)) throw json.error
```

Comment:
shouldn't we rather just fix this in the server, so return the previous `.error` field? then no need to patch the SDK.